package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@api-platform/api-doc-parser",
-  "version": "0.7.2",
+  "version": "0.7.3",
```

Comment:
Can you revert this? It's the CI that takes care of this.

Reply:
Great :+1: